src/bootstrap/bin/rustdoc.rs

```diff
@@ -58,9 +59,9 @@ fn main() {
     if env::var_os("RUSTDOC_FUSE_LD_LLD").is_some() {
         cmd.arg("-Clink-arg=-fuse-ld=lld");
         if cfg!(windows) {
-            cmd.arg("-Clink-arg=-Wl,/threads:1");
+            cmd.arg(format!("-Clink-arg=-Wl,{}", lld_flag_no_threads(true)));
         } else {
-            cmd.arg("-Clink-arg=-Wl,--threads=1");
+            cmd.arg(format!("-Clink-arg=-Wl,{}", lld_flag_no_threads(false)));
```

Comment:
It's better to do probing for LLD version only once in the `bootstrap` binary.
Then the result can be sent to the `src/bootstrap/bin/rustdoc.rs` wrapper through an environment variable, similarly to e.g. `RUSTDOC_FUSE_LD_LLD`.